src/storage/file_io.c

```diff
@@ -6323,6 +6323,13 @@ fileio_get_volume_label_by_fd (int vol_fd, bool is_peek)
   return fileio_get_volume_label (fileio_get_volume_id (vol_fd), is_peek);
 }
 
+const char *
+fileio_get_volume_label_with_unknown (VOLID volid)
+{
+  static constexpr char *UNKNOWN_VLABEL = "(UNKNOWN)";
+  const char *vlabel = fileio_get_volume_label (volid, PEEK);
```

Comment:
I would argue that the extra param is not needed. There are two usecases where the opposite of `PEEK` - `ALLOC_COPY` - is used, but they are not needed since the value is copied again where needed anyway (`logpb_delete` and `log_unformat_ahead_volumes`).
Therefore it can be removed. But perhaps not in the scope of this PR.
Also, `fileio_get_volume_label` can return `const char *`.

Reply:
Yeah, I thought of it too, but it was not in the scope of this PR. Without removing the `ALLOC_COPY` argument, `fileio_get_volume_label` cannot return `const char *`.